scipy/stats/tests/test_stats.py

```diff
@@ -5421,12 +5421,10 @@ def test_resampling_input_validation(self):
 
 @pytest.mark.filterwarnings("ignore:Arguments...:DeprecationWarning")
```

Comment:
I was wondering if this is still needed? (sorry didn't have time to check myself)